static/app/components/events/contexts/memoryInfo/getMemoryInfoKnownDataDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import {t} from 'sentry/locale';
+import {Event, MemoryInfoContext, MemoryInfoContextKey} from 'sentry/types/event';
+
+export const memoryInfoKnownDataValues = [...Object.values(MemoryInfoContextKey)];
+
+type Output = {
+  subject: string;
+  value: React.ReactNode | null;
+};
+
+type Props = {
+  data: MemoryInfoContext;
+  event: Event;
+  type: keyof typeof memoryInfoKnownDataValues;
+};
+
+export function getMemoryInfoKnownDataDetails({data, type}: Props): Output | undefined {
```

Comment:
Assuming you're getting these from data sent by the .NET SDK, there are a few more to consider.  Reference the source code here:

https://github.com/getsentry/sentry-dotnet/blob/01ca1d4e2e850d33984929668558068376c4aa2e/src/Sentry/Internal/MemoryInfo.cs#L81-L107

Thanks.